src/browser/ui/ReactDOMComponent.js

```diff
@@ -87,6 +88,12 @@ function checkAndWarnForMutatedStyle(style1, style2, component) {
  */
 var BackendIDOperations = null;
 
+var eventTypes = Object.keys(EventConstants.topLevelTypes)
+  .reduce(function(eventTypesAccumulator, key) {
+    var eventType = key.replace(/^top/, 'on');
+    eventTypesAccumulator[eventType.toLowerCase()] = eventType;
+    return eventTypesAccumulator;
+  }, {});
```

Comment:
I think we probably want this in a `__DEV__` block so we aren't doing extra startup work in prod (it might get dead code eliminated but not certain)